src/features.js

```diff
@@ -31,7 +32,8 @@ export const platformSupport = {
         ...baseFeatures
     ],
     'apple-isolated': [
-        'duckPlayer'
+        'duckPlayer',
+        'brokerProtection'
```

Comment:
does this affect iOS too?

Reply:
no, currently iOS is not loading apple-isolated, but even if it were, we obviously wouldn't enable there (via config)

Reply:
Will the feature code be bundled when iOS starts loading apple-isolated?